### ISSUE TITLE
Details on training and testing

### ISSUE THREAD

**Issue by Ankit2527** [davide-coccomini/Combining-EfficientNet-and-Vision-Transformers-for-Video-Deepfake-Detection#41]
Hi @davide-coccomini,
As you mentioned in the paper, that you have used both FF++ and DFDC dataset to train the model:
- Which version of FF++ dataset do you use (raw/c23/c40)?
- As for DFDC only the training data is available and the public and private test sets are not released. How do you use the DFDC dataset? Do you do a split from the training set itself into train/val/test?

**Comment by davide-coccomini** [comment]
Hi, the FF++ version used is c23.
The DFDC dataset, labels and splits can be found at: https://dfdc.ai/

**Comment by Ankit2527** [comment]
Thank you @davide-coccomini for your quick response!

**Comment by viethc02** [comment]
I got some errors when trying to download dfdc data. Have you downloaded it yet? I will be very grateful if you can share with me the test dataset via my email. Thank you so much.

**Comment by davide-coccomini** [comment]
Hi, what kind of errors are you getting?

**Comment by viethc02** [comment]
Exceeded daily email limit for the operation or the account. If a higher limit is required, please configure your user pool to use your own Amazon SES configuration for sending email.

Thanks for replying! I got that error when trying to register an account on dfdc.ai. I tried to fix it but still can't sign up and some of my friends said it was due to website error. Do you have any suggestions for me?

**Comment by davide-coccomini** [comment]
Unfortunately not, many people are having issues with DFDC dataset downloading but we can't do anything...
